.github/workflows/release.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+
 jobs:
   release:
     uses: boldlink/terraform-module-support/.github/workflows/release.yaml@1.0.0
```

Comment:
That is good @hugoalmeida264 , this will have to be changed from `terraform-module-template`  so that it can reflect in the next scheduled update for all repos.